boundary_safety:
    x_min: 0.3
    x_max: 0.6
    y_min: -0.5
    y_max: -0.05
    z_min: 0.005
    z_max: 0.1
